ci/terraform/oidc/ecc-signing-key.tf

```diff
@@ -64,8 +64,10 @@ data "aws_iam_policy_document" "storage_token_signing_key_access_policy" {
   }
 }
 
+//NOTE: This key is used to sign the storage token included as a claim inside the
+//MFA reset JAR.
 resource "aws_kms_key" "mfa_reset_signing_key_ecc" {
```

Comment:
Could rename this `mfa_reset_token_signing_key_ecc`?

Reply:
Did you mean something like `mfa_reset_storage_token_signing_key_ecc` ?

Reply:
Yep that works too 👍